home.html

```diff
@@ -31,56 +27,26 @@
     </style>
 </head>
 <body>
+            <!--Nav:With hero section-->
```

Comment:
Your navbar looks separate from your hero so do not make comment as if its together. Your navbar has a visible height which separates it from the hero section.